src/test/java/com/epam/tc/hw2/ex1/FirstExercise.java

```diff
@@ -0,0 +1,113 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BaseClass;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class FirstExercise extends BaseClass {
+
+    SoftAssertions softAssert = new SoftAssertions();
+
+    @Test
+    public void mainFirstTest() {
+        // 2. Assert Browser title
+        softAssert.assertThat(driver.getTitle()).isEqualTo("Home Page");
+        // 3. Perform login
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
```

Comment:
There is no need to open the same page several times.
`driver.get("https://jdi-testing.github.io/jdi-light/index.html");` is used in setUp method and two times in this test.

Reply:
Oops, accidentally